feat: policy violations filter (#2550)
chainloop-dev/chainloop: CLAUDE.md

- if you add any new dependency to a constructor, remember to run wire ./...
- when creating PR message, keep it high-level, what functionality was added, don't add info about testing, no icons, no info about how the message was generated.
- app/controlplane/api/gen/frontend/google/protobuf/descriptor.ts is a special case that we don't want to upgrade, so if it upgrades, put it back to main
- when creating a commit or PR message, NEVER add co-authored by or generated by Claude code

> - if you modify a schema, remember to run migration_diff in makefile or similar

Reviewer: Don't you think these instructions could be a bit ambiguous for an agent?
```suggestion
- if you modify a schema, remember to run `make migration_sync`
```
